XLS-37d/README.md

```diff
@@ -0,0 +1,202 @@
+```
```

Comment:
Folder name should be updated to reflect `XLS-0000d-{title}` where title is lowercase and spaces replaced by a hyphen.

Reply:
Do you mind if I shorten the titles? Or is `concise-transaction-identifier` what you're looking for?